common/keys.go

```diff
@@ -235,4 +235,7 @@ const (
 	// ArgoCDClusterConfigNamespacesEnvVar is the environment variable that contains the list of namespaces allowed to host cluster config
 	// instances
 	ArgoCDClusterConfigNamespacesEnvVar = "ARGOCD_CLUSTER_CONFIG_NAMESPACES"
+
+	// ArgoCDComponentStatus is the default group name of argocd-component-status-alert prometheusRule
+	ArgoCDComponentStatus = "ArgoCDComponentStatus"
```

Comment:
I think the comment needs to be updated and the value should be `argocd-component-status-alert`

Reply:
The name of this `prometheusRule` resource is `argocd-component-status-alert`, while the name of this Rule Group is `ArgoCDComponentStatus`. Please take a look at this document

https://github.com/argoproj-labs/argocd-operator/blob/operator-redesign/docs/usage/monitoring.md?plain=1#L41

Reply:
ah yeah, you're right, my bad
Although this should probably be in values.go, since this is a value not a key